frontend/packages/kubevirt-plugin/integration-tests/tests/utils/scripts/guest-agent-login.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/expect -f
```

Comment:
this file is very similar to `expect-file-exists.sh`, add a sleep in `expect-file-exists.sh` can achieve the purpose here.

Reply:
it is not similar because there is additional functionality in the script that do a completely different thing (that is: checking for pre-uploaded file. this is not what *this* test is about.